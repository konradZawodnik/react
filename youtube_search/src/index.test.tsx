import YoutubeSearchContainer from "./components/YoutubeSearchContainer";

jest.mock("react-dom/client", () => ({
  createRoot: jest.fn().mockImplementation(() => ({
    render: jest.fn(),
  })),
}));

describe("Application root", () => {
  it("Should render app without crashing", () => {
    expect(
      JSON.stringify(
        Object.assign({}, YoutubeSearchContainer, {
          _reactInternalInstance: "censored",
        }),
      ),
    ).toMatchSnapshot();
  });
});
