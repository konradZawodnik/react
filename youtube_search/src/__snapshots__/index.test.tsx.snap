// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Application root Should render app without crashing 1`] = `"{\\"_reactInternalInstance\\":\\"censored\\"}"`;
